src/helpers/formatPool.ts

```diff
@@ -0,0 +1,58 @@
+import { getAssetCode } from "helpers/getAssetCode";
```

Comment:
the "all pools" endpoint is supposedly going to be structured the same way as individual pools, so creating this helper to keep our redux format in sync